builder-20/builder.toml

```diff
@@ -6,7 +6,7 @@ build-image = "heroku/heroku:20-cnb-build"
 run-image = "heroku/heroku:20-cnb"
 
 [lifecycle]
-version = "0.18.5"
+version = "0.19.0-rc.2"
```

Comment:
The versions in the table in `README.md` needs updating to match too.

Reply:
Thanks, done. https://github.com/heroku/cnb-builder-images/pull/472/commits/b70d4391f9796efb5b68d0f5b86d2344e15fdcf8